models/snapchat__campaign_report.sql

```diff
@@ -1,21 +1,50 @@
-with adapter as (
+with campaign_hourly as (
 
     select *
-    from {{ ref('snapchat__ad_adapter') }}
+    from {{ var('campaign_hourly_report') }}
+
+), account as (
+
+    select *
+    from {{ var('ad_account_history') }}
+    where is_most_recent_record = true
+
+), ad_squads as (
+
+    select *
+    from {{ var('ad_squad_history') }}
+    where is_most_recent_record = true
```

Comment:
Do we need to to bring ad_squads into this model? It doesn't look like we use it in the final cte and the join isn't required to get the ad account info.